packages/in_app_purchase/in_app_purchase_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.4+8
+
+* Fix broken link in packages/in_app_purchase/in_app_purchase_android/README.
```

Comment:
Since the CHANGELOG is already part of the `packages/in_app_purchase/in_app_purchase_android` folder you can leave this out of the comment as it is more difficult to read and might confuse users. The following should suffice:
```suggestion
* Fix a broken link in the README.
```